setup.cfg

```diff
@@ -0,0 +1,2 @@
+[aliases]
+test=pytest
```

Comment:
The red symbol at the end here means this file doesn't end with a newline.  Can you make sure that all of these files end in newlines?